package.json

```diff
@@ -16,17 +16,23 @@
 		"@babel/plugin-proposal-class-properties": "^7.1.0",
 		"@babel/preset-env": "^7.1.6",
 		"@babel/preset-react": "^7.0.0",
+		"axios": "^0.18.0",
```

Comment:
why do you have axios dependencies when we are using fetch .?
Can you kindly  uninstall those from the package.json

Reply:
@bozicschucky I have remove it